sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -1104,10 +1104,10 @@ class SessionCatalog(
    */
   def registerFunction(
```

Comment:
@ouyangxiaochen , you can call this method in https://github.com/apache/spark/pull/17681/files#diff-d2a203f08c862bd762e6740c16e972f7R84